pom.xml

```diff
@@ -131,11 +131,6 @@
           <artifactId>maven-project-info-reports-plugin</artifactId>
           <version>3.4.5</version>
         </plugin>
-        <plugin>
-          <groupId>org.sonatype.plugins</groupId>
-          <artifactId>nexus-staging-maven-plugin</artifactId>
-          <version>1.6.13</version>
-        </plugin>
```

Comment:
`camunda-bpm-release-parent` uses the `nexus-staging-maven-plugin` in version 1.6.13.